src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void  doWork() {
```

Comment:
There is an extra space between `void` and `doWork`. It should be `public void doWork()`.